rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2397,6 +2397,42 @@ void voteFederationChange_addFederatorPublicKeyWithLessThanMofNVotes_returnsSucc
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorPublicKey_returnsExpectedPendingFederationSizeResponseCode() {
```

Comment:
`_returnsSuccessful`. The method is not returning any federation size response code